src/i18n/rpi-imager_it.ts

```diff
@@ -216,64 +216,64 @@ Aggiungi sia &apos;rpi-imager.exe&apos; che &apos;fat32format.exe&apos; all&apos
     <message>
         <location filename="../DstPopup.qml" line="24"/>
         <source>Storage</source>
-        <translation type="unfinished">Scheda SD</translation>
+        <translation>Storage</translation>
     </message>
     <message>
         <location filename="../DstPopup.qml" line="38"/>
         <source>No storage devices found</source>
-        <translation type="unfinished">Nessun dispositivo archiviazione trovato</translation>
+        <translation>Nessun dispositivo archiviazione trovato</translation>
     </message>
     <message>
         <location filename="../DstPopup.qml" line="67"/>
         <source>Exclude System Drives</source>
-        <translation type="unfinished"></translation>
+        <translation>Escludi unità sistema</translation>
     </message>
     <message>
         <location filename="../DstPopup.qml" line="94"/>
         <source>gigabytes</source>
-        <translation type="unfinished"></translation>
+        <translation>Gigabyte</translation>
     </message>
     <message>
         <location filename="../DstPopup.qml" line="96"/>
         <location filename="../DstPopup.qml" line="152"/>
         <source>Mounted as %1</source>
-        <translation type="unfinished">Montato come %1</translation>
+        <translation>Montato come %1</translation>
     </message>
     <message>
         <location filename="../DstPopup.qml" line="139"/>
         <source>GB</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```

Comment:
An empty translation string doesn't seem right?! :thinking:

Reply:
I didn't fin untranslated strings in my Italian uploaded file